### ISSUE TITLE
Withdraw funds from on-chain wallet

### ISSUE THREAD

**Issue by holzeis** [itchysats/10101#114]
Like the title suggest, the possibility to withdraw funds from the wallet.

![Image](https://user-images.githubusercontent.com/382048/199558583-9b849268-78eb-4394-b2e1-c1b41547adcf.png)

**Comment by holzeis** [comment]
I guess withdrawing will also automatically mean closing the lightning channel? Alternatively we could ask the user for what to do.

**Comment by bonomat** [comment]
> I guess withdrawing will also automatically mean closing the lightning channel? Alternatively we could ask the user for what to do.

Depends on if the money is off-chain or on-chain. Withdrawing the off-chain funds can be done with a on-chain/off-chain swap, i.e. they shift all the funds to a service provider who in return sends the funds on-chain to an address their choice. 
However, given the user always has a channel opened with us, the channel won't be of much use for us after the swap so it's likely we would close it :)

Long story short. I don't think we need a particular `withdraw` functionality, but a simple `send funds on-chain` and `send funds off-chain` (aka pay invoice). With this they user can withdraw all their funds to somewhere else.

**Comment by bonomat** [comment]
It would be cool if we could model a generic `Send` screen which is able to distinguish between a lightning invoice QR code and a bitcoin on-chain address QR code.
